README.md

```diff
@@ -127,33 +92,35 @@ openstack endpoint create logs_v2 public http://192.168.10.6:5607/v2.0
 
 ## Running the installer
 
+Download the installer from [here](https://github.com/monasca/monasca-agents-installer/releases)
+
 ### Metrics agent
 Please use the embedded help for detailed and up-to-date info:
 
 ```
-./monasca-agent-<version_number>.run --help
+./monasca-agent-<version>.run --help
 ```
 
 To provide Keystone credentials and configure the agent using auto-detection run the following command:
 
 ```
-./monasca-agent.run --target /opt/monasca/monasca-agent -- --username <username> --password <password>\
+./monasca-agent-<version>.run --target /opt/monasca/monasca-agent -- --username <username> --password <password>\
                     --project_name <project> --keystone_url <keystone_url>
 ```
-This will create and run a new service file `/etc/systemd/system/monasca-agent.service` with the configuration set as per the arguments mentioned above. 
+This will create and run a new service file `/etc/systemd/system/monasca-agent.service` with the configuration set as per the arguments mentioned above.
 
 ### Log agent
 Please use the embedded help for detailed and up-to-date info:
 ```
-./log-agent-<logstash_version>_<logstash_output_monsaca_log_api_version>.run --help
+./log-agent-<version>.run --help
```

Comment:
Just to confirm, here: do you want to remove the user's ability to specify a logstash version at the time of creating the log-agent installer? Or do you just want to reduce the name of the .run file to not include the logstash version?

Reply:
You have to distinguish between user and developer. It is ok that a developer (you) to specify the logstash version. However, does it makes sense for users to include library version numbers into monasca artifact names? What happens if you fix the shell scritps and create a new download package? Do you keep the same version number? Sounds very strange to me. In my understanding the versioning should be aligned with monasca version. How should a user know what to download otherwise?